### PR TITLE
feat(cspr): update CSPR explorer URLs

### PR DESCRIPTION
**Pull request by Krin2**
Changing on account and explorer URLs for CSPR

STLX-1657

https://bitgoinc.atlassian.net/browse/STLX-1657

### CODE REVIEW

File: modules/statics/src/networks.ts [hunk 1]
```diff
@@ -204,15 +204,17 @@ class CeloTestnet extends Testnet implements EthereumNetwork {
 //TODO update explorerUrl STLX-1657
 class Casper extends Mainnet implements AccountNetwork {
   family = CoinFamily.CSPR;
-  explorerUrl = 'https://clarity.casperlabs.io/#/deploys/';
-  accountExplorerUrl = '';
+  explorerUrl = 'https://cspr.live/deploy/ ';
+  accountExplorerUrl = 'https://cspr.live/account/ ';
+
+
```
Comment (smaria95) [inline modules/statics/src/networks.ts:210]: Remove this extra lines

File: modules/statics/src/networks.ts [hunk 2]
```diff
@@ -204,15 +204,17 @@ class CeloTestnet extends Testnet implements EthereumNetwork {
...
 }
 
 //TODO update explorerUrl STLX-1657
 class CasperTestnet extends Testnet implements AccountNetwork {
   family = CoinFamily.CSPR;
-  explorerUrl = 'https://clarity.casperlabs.io/#/deploys/';
-  accountExplorerUrl = '';
+  explorerUrl = 'https://testnet.cspr.live/deploy/ ';
```
Comment (smaria95) [inline modules/statics/src/networks.ts:216]: Please remove the extra space at the end of this string.

**Review by JulioAchaval-SimTLiX — DISMISSED** [review]
LGTM

**Review by JulioAchaval-SimTLiX — DISMISSED** [review]
LGTM

**Review by CapnMigraine — APPROVED** [review]
looks good

**Review by rliberal-simtlix — DISMISSED** [review]

**Review by tylerlevine — CHANGES_REQUESTED** [review]
drone is failing, I think you need to rebase

**Review by rliberal-simtlix — APPROVED** [review]

**Review by CapnMigraine — APPROVED** [review]

**Review by rliberal-simtlix — APPROVED** [review]

**Review by tylerlevine — APPROVED** [review]

**Review by JulioAchaval-SimTLiX — APPROVED** [review]

**Review by smaria95 — APPROVED** [review]
LGTM

**Pull request merged**